Makefile

```
@@ -689,7 +689,7 @@ APIDIFF_OLD_COMMIT ?= $(shell git rev-parse origin/main)
 apidiff: $(GO_APIDIFF) ## Check for API differences
 	$(GO_APIDIFF) $(APIDIFF_OLD_COMMIT) --print-compatible
 
-ALL_VERIFY_CHECKS = licenses boilerplate shellcheck tiltfile modules gen conversions doctoc capi-book-summary diagrams import-restrictions go-directive
+ALL_VERIFY_CHECKS = licenses boilerplate shellcheck tiltfile modules gen conversions doctoc capi-book-summary diagrams import-restrictions go-directive security
```

```suggestion
ALL_VERIFY_CHECKS = licenses boilerplate shellcheck tiltfile modules gen conversions doctoc capi-book-summary diagrams import-restrictions go-directive govulncheck
```
Let's not add all the security check for now, because verify-container-images includes docker-build that takes a few minutes (we should keep PR related jobs as fast as possible) 

Eventually what we can do is to add an optional job to run trivy, but TBH I don't think it is necessary

---

Thanks. Fixed.